zsbl/ux00_zsbl.dts

```diff
@@ -152,43 +148,15 @@
 	L55: soc {
 		#address-cells = <2>;
 		#size-cells = <2>;
-		compatible = "freechips,rocketchip-unknown-soc", "simple-bus";
+		compatible = "SiFive,FU540G-soc", "fu500-soc", "sifive-soc", "simple-bus";
 		ranges;
-		L8: bus-error-unit@1700000 {
-			compatible = "sifive,buserror0";
-			interrupt-parent = <&L2>;
-			interrupts = <55>;
-			reg = <0x0 0x1700000 0x0 0x1000>;
-			reg-names = "control";
-		};
-		L12: bus-error-unit@1701000 {
-			compatible = "sifive,buserror0";
-			interrupt-parent = <&L2>;
-			interrupts = <56>;
-			reg = <0x0 0x1701000 0x0 0x1000>;
-			reg-names = "control";
-		};
-		L16: bus-error-unit@1702000 {
-			compatible = "sifive,buserror0";
-			interrupt-parent = <&L2>;
-			interrupts = <57>;
-			reg = <0x0 0x1702000 0x0 0x1000>;
-			reg-names = "control";
-		};
-		L20: bus-error-unit@1703000 {
-			compatible = "sifive,buserror0";
-			interrupt-parent = <&L2>;
-			interrupts = <58>;
-			reg = <0x0 0x1703000 0x0 0x1000>;
-			reg-names = "control";
-		};
-		L24: bus-error-unit@1704000 {
-			compatible = "sifive,buserror0";
-			interrupt-parent = <&L2>;
-			interrupts = <59>;
-			reg = <0x0 0x1704000 0x0 0x1000>;
```

Comment:
@tmagik All of these changes made to the zsbl.dts are actually correct. The DTS checked into this repository reflects what the U540 would look like if we taped it out again today. However, at the time RTL was finalized, things like the bus-error-unit and the 3rd L2 interrupt did not yet exist. If this repository is intended to reflect the shipped part, we should fix the DTS.

Reply:
Yes, that's why I replied to @tmagic

> Btw. what are the plans with the freedom-u540-c000-bootloader
> repository? Are you going to continue development on it and reuse new
> versions in other chips, or is it more like a point-in-time code-dump?
> If it's the first then I'll refactor my pull request a bit so that
> it's easier to keep the "frozen" code for the zsbl burnt into this
> first chip separate and continue joint development on both the zsbl
> and fsbl for newer chips.

..but I never received any reply before now.